RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletGeneratorKernelsImpl.h

```diff
@@ -376,9 +379,10 @@ __global__ void kernel_mark_used(GPUCACell *__restrict__ cells, uint32_t const *
   }
 }
 
-__global__ void kernel_countMultiplicity(HitContainer const *__restrict__ foundNtuplets,
-                                         Quality const *__restrict__ quality,
-                                         caConstants::TupleMultiplicity *tupleMultiplicity) {
+template <CountOrFill cof>
```

Comment:
I would prefer
```suggestion
template <CountOrFill COF>
```
or
```suggestion
template <CountOrFill CASE>
```

Reply:
CoF ?

Reply:
Thinking back I like `CASE` best, because it is used like `if constexpr (CountOrFill::count == CASE)` .

But I don't think we have any "rules" about naming template parameters, so it can also stay as `cof`.